source/includes/_first-things-first.md

```diff
@@ -28,7 +28,7 @@ The API follows a typical RESTful pattern where it allows you to access collecti
 Some objects returned by the API may have sub-collections. For example, a tags collection for a given user object. These sub-collections are paginated in the same manner as top level collections.
 
 ```
-curl -X GET https://api.10000ft.com/api/v1/users \
+curl -X GET https://api.rm.smartsheet.com/api/v1/users \
```

Comment:
In other places you've changed `https` to `http`, does it need to be `http` here too?  Same for the other places in this PR

Reply:
OMG!!! 🙈  Good catch, the protocol should not change and should be set to 'https'. Let me update this.